.github/workflows/owasp-check.yml

```
@@ -0,0 +1,40 @@
+name: "OWASP Dependency Check"
+
+on:
+
```

```suggestion
  workflow_dispatch:
```

To be able to trigger it manually as well, if we ever need it